client/auth/index.js

```diff
@@ -11,13 +11,14 @@ import page from 'page';
  */
 import config from 'config';
 import controller from './controller';
+import { makeLayout, render as clientRender } from 'controller';
 
 export default () => {
 	// Always enable the /oauth-login route and redirect to /log-in if `oauth` is disabled
-	page( '/oauth-login', controller.oauthLogin );
+	page( '/oauth-login', controller.oauthLogin, makeLayout, clientRender );
 
 	if ( config.isEnabled( 'oauth' ) ) {
-		page( '/authorize', controller.authorize );
-		page( '/api/oauth/token', controller.getToken );
+		page( '/authorize', controller.authorize, makeLayout, clientRender );
+		page( '/api/oauth/token', controller.getToken, makeLayout, clientRender );
```

Comment:
We need to make sure that for each section's route definitions (normally found in its `index.js`) that is changed by the codemod, the corresponding controllers (normally in `controller.js`) are changed as well (and vice versa!).

(In this case, the `oathLogin`, `authorize`, and `getToken` controllers.)

Reply:
PR #19567